ui/src/shared/components/CustomTimeRange.js

```diff
@@ -8,6 +8,7 @@ class CustomTimeRange extends Component {
 
     this.handleClick = ::this.handleClick
     this._formatTimeRange = ::this._formatTimeRange
+    this.handleMovingTimeRange = ::this.handleMovingTimeRange
```

Comment:
this should not be called `handleMovingTimeRange` if this is not about moving time ranges